msal-python-conceptual/advanced/macos-broker.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Using MSAL Python with Authentication Brokers on macOS
+description: "Using authentication brokers on macOS enables you to simplify how your users authenticate with Microsoft Entra ID from your application, as well as take advantage of advanced functionality such as token binding, protecting any issued tokens from exfiltration and misuse."
+author: localden
+manager: CelesteDG
+
+ms.service: msal
+ms.subservice: msal-python
+ms.topic: conceptual
+ms.date: 02/07/2024
```

Comment:
I know where this date came from. It is a copy-and-paste error from the pre-existing `wam.md` doc. :-)

On a more serious note, we could also reuse the same `wam.md` doc by adding a new "Prerequisite of using broker on macOS" section. Besides the different prerequisites and different opt-in feature flag, the code snippets on both platforms are the same.